docs/csharp/event-pattern.md

```diff
@@ -173,18 +142,7 @@ indicate a cancel incorrectly.
 Let's implement the first version for this sample. You need to add a
 boolean field to the FileFoundEventArgs type:
 
-```csharp
-public class FileFoundArgs : EventArgs
-{
-    public string FoundFile { get; }
-    public bool CancelRequested { get; set; }
-
-    public FileFoundArgs(string fileName)
-    {
-        FoundFile = fileName;
-    }
-}
-```
+[!code-csharp[EventArgs](../../samples/csharp/events/Program.cs#EventArgs "Update event arguments")]
 
 This new Field should be initialized to false, so you don't cancel
```

Comment:
NIT: This new Field should be initialized to false, so you don't cancel **for no reason.** <- accidentally?

Reply:
Actually, the field isn't (explicitly) initialized, which makes this confusing. Better is "This field is automatically initialized to \`false\`, the default value of a Boolean, so you don't cancel accidentally."

also line 167 below: request a cancel --> request a cancellation